### PR TITLE
[NEW] Implement commands.run REST endpoint call

### PR DESCRIPTION
**Pull request by leonardoaramaki**
https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/pull/87 should be merged first before this one.

### CODE REVIEW

File: core/src/test/kotlin/chat/rocket/core/internal/rest/CommandsTest.kt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+            with(commands.result[21]) {
+                assertThat(command, isEqualTo("unarchive"))
+                assertThat(clientOnly, isEqualTo(false))
+            }
+        }
+    }
+
+    @Test
+    fun `runCommand() should signal the backend to run the specified command`() {
```
Comment (rafaelks) [inline core/src/test/kotlin/chat/rocket/core/internal/rest/CommandsTest.kt]: Maybe you could also add a fail test case here?

File: core/src/test/kotlin/chat/rocket/core/internal/rest/CommandsTest.kt [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            tokenRepository = this@CommandsTest.tokenProvider
+            platformLogger = PlatformLogger.NoOpLogger()
+        }
+
+        Mockito.`when`(tokenProvider.get()).thenReturn(authToken)
+    }
+
+    @Test
+    fun `commands() should return a list of available commands on a server`() {
+        mockServer.expect()
```
Comment (rafaelks) [inline core/src/test/kotlin/chat/rocket/core/internal/rest/CommandsTest.kt]: Maybe you could also add a fail test case here also?

**Comment by leonardoaramaki** [comment]
@luciofm @rafaelks @filipedelimabrito How should we proceed on this matter of testing for failures for now?

**Comment by rafaelks** [comment]
@leonardoaramaki I would say we can leave it for now.

**Pull request merged**